libs/blocks/images/images.js

```diff
@@ -0,0 +1,21 @@
+import { createTag } from '../../utils/utils.js';
+
+/**
+* Initializes the image block by wrapping each picture element in a div with a class of "figure"
+*@param {Element} blockEl - The block element to initialize.
+*/
+export default function init(blockEl) {
```

Comment:
can you run npm run lint on this file and fix the linting errors? 
On line 11 you're missing a semicolon after your return statement for example.